nps-unit-sample.json

```diff
@@ -1,367 +1,367 @@
-        },
-        {
-          "year": 2001,
-          "conditionStatus": "Warrants Moderate Concern",
-          "conditionTrend": "Relatively Unchanged",
-          "conditionConfidence": "High Confidence",
-          "conditionSummaryCombined": "concern-unchanging-high",
-          "benchmarkChartYAxisLabel": "Haze Index, dv",
-          "benchmarkChartValue": 11.5,
-          "benchmarkChartConcern": 15,
-          "benchmarkChartAverage": 12,
-          "benchmarkChartGood": 9,
-          "benchmarkChartColor": "Yellow",
-          "rationaleCondition": "Visibility warrants moderate concern at Acadia NP. This status is based on <a href='https://www.nps.gov/subjects/air/analysis-methods.htm'>NPS Air Resources Division benchmarks</a> and the 2011–2015 estimated visibility on mid-range days of 11.6 deciviews (dv), which is 4.4 dv above the estimated natural condition of 7.2 dv.",
-          "rationaleTrend": "For 2006–2015, the trend in visibility at Acadia NP improved on the 20% clearest and 20% haziest days, resulting in an overall improving visibility trend (IMPROVE Monitor ID: ACAD1, ME). The Clean Air Act visibility goal requires visibility improvement on the 20% haziest days, with no degradation on the 20% clearest days.",
-          "rationaleConfidence": "The degree of confidence at Acadia NP is is high because there is an on-site or nearby visibility monitor."
-        },
-        {
-          "year": 2002,
-          "conditionStatus": "Warrants Moderate Concern",
-          "conditionTrend": "Relatively Unchanged",
-          "conditionConfidence": "High Confidence",
-          "conditionSummaryCombined": "concern-unchanging-high",
-          "benchmarkChartYAxisLabel": "Haze Index, dv",
-          "benchmarkChartValue": 11.5,
-          "benchmarkChartConcern": 15,
-          "benchmarkChartAverage": 12,
-          "benchmarkChartGood": 9,
-          "benchmarkChartColor": "Yellow",
-          "rationaleCondition": "Visibility warrants moderate concern at Acadia NP. This status is based on <a href='https://www.nps.gov/subjects/air/analysis-methods.htm'>NPS Air Resources Division benchmarks</a> and the 2011–2015 estimated visibility on mid-range days of 11.6 deciviews (dv), which is 4.4 dv above the estimated natural condition of 7.2 dv.",
-          "rationaleTrend": "For 2006–2015, the trend in visibility at Acadia NP improved on the 20% clearest and 20% haziest days, resulting in an overall improving visibility trend (IMPROVE Monitor ID: ACAD1, ME). The Clean Air Act visibility goal requires visibility improvement on the 20% haziest days, with no degradation on the 20% clearest days.",
-          "rationaleConfidence": "The degree of confidence at Acadia NP is is high because there is an on-site or nearby visibility monitor."
-        }
-      ],
-      "monitoringSites": [
-        {
-          "name":"IMPROVE: ACAD1",
-          "trendCharts": [
-            {
```

Comment:
![screen shot 2018-11-09 at 1 02 02 pm](https://user-images.githubusercontent.com/3201135/48279809-ae0ce580-e41f-11e8-97f8-10cb5dba0063.png)
This is all the data needed to build this chart.

As the user changes the end year and time-period, the front end will narrow down the window of which `annualValues` it draws along with which `trendLines` to draw. I'm assuming the `constantLines` are the same across years and don't need their own `annualValues`?

Reply:
Yes, I think this will work.

Reply:
The combined haziest and clearest days charts don't include trend lines. Is it okay if the trendLines array is empty for this chart type?

Reply:
@bdavisnoland Yep, that works!

Reply:
Per discussion 11/14: There should be a shared name across all charts for the trend line.

Reply:
Added in https://github.com/vcavallo/unit-file-sample/pull/1/commits/6878da6852e2e4fe246c1da4fc346b0be1cc79eb